docs/streams.html

```diff
@@ -836,6 +836,32 @@
           <li> <code>value.serde</code> was deprecated and replaced by <code>default.value.serde</code>
         </ul>
 
+        <p>
+        Deprecated methods in <code>KTable</code>
+        </p>
+        <ul>
+            <li><code>void foreach(final ForeachAction&lt;? super K, ? super V&gt; action)</code></li>
+            <li><code>void print()</code></li>
+            <li><code>void print(final String streamName)</code></li>
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+            <li><code>void print(final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde, final String streamName)</code></li>
+            <li><code>void writeAsText(final String filePath)</code></li>
+            <li><code>void writeAsText(final String filePath, final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+            <li><code>void writeAsText(final String filePath, final String streamName)</code></li>
+            <li><code>void writeAsText(final String filePath, final String streamName, final Serde&lt;K&gt; keySerde, final Serde&lt;V&gt; valSerde)</code></li>
+        </ul>
+
+        <p>
+             These methods have been deprecated in favor of using the Interactive Queries API.  
```

Comment:
We should not say it is `the equivalent functionality`: the deprecated function is for printing the changelog stream of the KTable, and can still be executed by table.toStream.XX.

We can re-word the IQ part as If you want to query the current content of the state store backing the KTables, use the following approach ...